AllReadyApp/Web-App/AllReady/DataAccess/IAllReadyDataAccess.cs

```diff
@@ -8,72 +9,81 @@ public interface IAllReadyDataAccess
 
         #region Event CRUD
         IEnumerable<Event> Events { get; }
-        Event GetEvent(int eventId);
-        int GetManagingOrganizationId(int eventId);
-        IEnumerable<EventSignup> GetEventSignups(string userId);
-        IEnumerable<EventSignup> GetEventSignups(int eventId, string userId);
-        Task UpdateEvent(Event value);
-        IEnumerable<Event> EventsByPostalCode(string postalCode, int distance);
-        IEnumerable<Event> EventsByGeography(double latitude, double longitude, int distance);
-        IEnumerable<Resource> GetResourcesByCategory(string category);
+        Event GetEvent( int eventId );
```

Comment:
Can you check your IDE settings? It seems like something is adding spacing around parens, which  will change a lot of lines of code and make it difficult to spot the changes for a reviewer. Thanks!